src/app/gateways/views/BuiltinGatewayDetailView.vue

```diff
@@ -1,116 +1,259 @@
 <template>
-  <RouteView
-    v-slot="{ route, t }"
-    name="builtin-gateway-detail-view"
-    :params="{
-      mesh: '',
-      gateway: '',
-      codeSearch: '',
-      codeFilter: false,
-      codeRegExp: false,
-      dataplane: '',
-    }"
+  <DataSource
+    v-slot="{ data: me }: MeSource"
+    src="/me"
   >
-    <AppView>
-      <DataSource
-        v-slot="{ data, error }: MeshGatewaySource"
-        :src="`/meshes/${route.params.mesh}/mesh-gateways/${route.params.gateway}`"
-      >
-        <ErrorBlock
-          v-if="error"
-          :error="error"
-        />
-
-        <LoadingBlock v-else-if="data === undefined" />
-
-        <div
-          v-else
-          class="stack"
+    <RouteView
+      v-if="me"
+      v-slot="{ can, route, t }"
+      name="builtin-gateway-detail-view"
+      :params="{
+        mesh: '',
+        gateway: '',
+        codeSearch: '',
+        codeFilter: false,
+        codeRegExp: false,
+        page: 1,
+        size: me.pageSize,
+        query: '',
+        s: '',
+        dataPlane: '',
```

Comment:
nit: pretty sure we don't need all of these params here, we only should need the ones that are dependencies of this `*View.vue`